pkg/controller/issueapi/issue.go

```diff
@@ -99,14 +101,58 @@ func (ic *IssueAPI) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 
 	expiryTime := time.Now().Add(ic.config.GetVerificationCodeDuration())
 
+	var issuingUserID uint
+	var issuingAuthorizedAppID uint
+
+	// Attempt to find the authorized app.
+	rawAuthorizedApp, ok := httpcontext.GetOk(r, "authorizedApp")
```

Comment:
I would move this out to a utility method - I Think it may end up be using later

authUser, authApp, err := getAuthorizationFromContext()